server/pulp/server/async/app.py

```diff
@@ -137,8 +144,8 @@ def get_resource_manager_lock(name):
                 msg = _("Resource manager '%(name)s' attempted to acquire the the resource manager "
```

Comment:
Can a similar change be made to this logging statement.

`Hot spare pulp_resource_manager instance '%(resource_manager_name)s' detected.`